test_challenger.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+import pytest
+
+from sdp import challenger_sdp
```

Comment:
Would you mind just testing every/all .py files that have the functions “sliding_dot_product” and “setup”? Got from pmin=3 to pmax=13 (inclusive)

Reply:
@seanlaw 
Created PR #14. Will update it to address your comment.